### PR TITLE
Fix PotionAPI ignores icon flag

### PR DESCRIPTION
**Pull request by notTamion**
Fixes LivingEntity#getPotionEffect and LivingEntity#getActivePotionEffects not passing the icon flag in the PotionEffect ctor

### CODE REVIEW

File: patches/server/0396-Fix-PotionEffect-ignores-icon-flag.patch [hunk 1]
```diff
@@ -3,9 +3,10 @@ From: kickash32 <kickash32@gmail.com>
 Date: Fri, 8 May 2020 00:49:18 -0400
 Subject: [PATCH] Fix PotionEffect ignores icon flag
 
+Co-Authored-By: Tamion <70228790+notTamion@users.noreply.github.com>
```
Comment (Warriorrrr) [inline patches/server/0396-Fix-PotionEffect-ignores-icon-flag.patch]:
```suggestion
Co-authored-by: Tamion <70228790+notTamion@users.noreply.github.com>
```

**Review by Warriorrrr — COMMENTED** [review]
CraftPotionUtil#toBukkit seems to also still be using the old constructor which could be added aswell here.

**Review by lynxplay — COMMENTED** [review]
What about CraftPotionUtil#toBukkit ?
Would probably fit nicely into this PR too.

edit: Hi Warrior xD

**Review by Warriorrrr — APPROVED** [review]

**Comment by notTamion** [comment]
Not sure if i should have also added a variable but adding paper comments around those two lines just seemed kinda ugly. Let me know what you think

**Review by lynxplay — APPROVED** [review]

**Comment by notTamion** [comment]
should be all now (i hope)

File: patches/server/0396-Fix-PotionEffect-ignores-icon-flag.patch [hunk 2]
```diff
@@ -17,3 +18,43 @@ index 9ba85d099d0e6893dc3489ce0753119348e937b3..71a2ac37046adb3826bc2e76b4ac385c
          return true;
      }
  
+@@ -464,7 +464,7 @@ public class CraftLivingEntity extends CraftEntity implements LivingEntity {
+     @Override
+     public PotionEffect getPotionEffect(PotionEffectType type) {
+         MobEffectInstance handle = this.getHandle().getEffect(CraftPotionEffectType.bukkitToMinecraft(type));
+-        return (handle == null) ? null : new PotionEffect(CraftPotionEffectType.minecraftToBukkit(handle.getEffect()), handle.getDuration(), handle.getAmplifier(), handle.isAmbient(), handle.isVisible());
++        return (handle == null) ? null : new PotionEffect(CraftPotionEffectType.minecraftToBukkit(handle.getEffect()), handle.getDuration(), handle.getAmplifier(), handle.isAmbient(), handle.isVisible(), handle.showIcon()); // Paper
```
Comment (Lulu13022002) [inline patches/server/0396-Fix-PotionEffect-ignores-icon-flag.patch]: Now that you fixed the util you can use them (CraftPotionUtil#toBukkit, the class is already imported too), same below, above and in your other pr if this one is merged before the other

**Comment by notTamion** [comment]
not sure if i did this correctly that modifying was confusing. Could you look over that?

**Review by Machine-Maker — APPROVED** [review]

**Pull request merged**